### ISSUE TITLE
Mailer generator problems found

### ISSUE THREAD

**Issue by hundrog** [techgurupezza/devise_materialize#7]
**Mailer views with errors**
I just run the generator and the mailer have some errors

**To Reproduce**
I don't know if is doing in other scenarios 

**Expected behavior**
`<div clas"divider"></div>`
should be
`<div class="divider"></div>`

All the links are in this format
```
link_to 'Confirm my Account', 
        class: 'btn',
        confirmation_url(@resource, confirmation_token: @token)
```
And should be like this
```
link_to 'Confirm my Account,
         confirmation_url(@resource, confirmation_token: @token),
         class: 'btn'
```
with the class after the path

**Desktop (please complete the following information):**

- OS: [Windows 10 - WSL Ubuntu 18.04]
- Browser [chrome]
- Version [75.0.3770.142]

**Comment by chiefpansancolt** [comment]
Thanks @hundrog! I will look at updating today and release a new version ASAP Thank you for the find!

**Comment by chiefpansancolt** [comment]
Released apart of version 1.2.1
